worker/peergrouper/desired.go

```diff
@@ -53,8 +59,6 @@ func desiredPeerGroup(info *peerGroupInfo) ([]replicaset.Member, map[*machineTra
 	// TODO There are some other possibilities
 	// for what to do in that case.
 	// 1) leave them untouched, but deal
-	// with others as usual "i didn't see that bit"
-	// 2) leave them untouched, deal with others,
 	// but make sure the extras aren't eligible to
```

Comment:
I think this comment needs re-wording now.

Reply:
I think that was just accidental deletion, thanks for the catch.